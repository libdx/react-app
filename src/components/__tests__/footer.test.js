import React from 'react'
import { shallow } from 'enzyme'
import Footer from '../footer'

describe('Footer', () => {

    const brand = 'Brand'
    let component

    beforeEach(() => {
        component = shallow(<Footer brand={brand} />)
    })

    it('has brand label', () => {
        const span = component.find('span') 

        expect(span.exists()).toBe(true)
    })
})
