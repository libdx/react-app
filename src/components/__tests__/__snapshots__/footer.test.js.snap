// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="footer"
>
  <span
    className="brand-label"
  >
    Brand
  </span>
</div>
`;
